website/content/maintainers/community/contributor-growth-framework/long-term-contributors.md

```diff
@@ -3,7 +3,6 @@ title: "Long-term contributors"
 linkTitle: "Long-term Contributors"
 date: 2021-04-8
 weight: 6
-draft: true
 ---
 
 But you also want more permanent non-code contributions. Think tech writers, PXMs, or product managers, all have a great skill set your project can benefit from. Unfortunately, most of these people don't really know that they, too, can be active members of the open source community.  
```

Comment:
`Think tech writers, PXMs, or product managers` - QA folks as well? Or those whose focus is mostly on testing/validating the project, in non-automated fashion.